specifications/dcp.profiles.md

```diff
@@ -0,0 +1,33 @@
+# Profiles of the Decentralized Claims Protocol
+
+## Introduction
+
+Many different variations of the VerifiableCredentials data model, the revocation system and the proof stack exist which
+makes it almost impossible to reach a sufficient level of interoperability. Profiles of the DCP specification help
+narrow down those possibilities.
+
+The DCP working group defines two basic profiles, each of which is referenced by an alias.
+
+## DCP profile definitions
+
+| alias             | VC data model                                               | revocation system                                                        | proof stack                                                                 | remarks                                                                                                                                                                                                           |
+|-------------------|-------------------------------------------------------------|--------------------------------------------------------------------------|-----------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `vc20-bssl/jwt`   | [VC DataModel 2.0](https://www.w3.org/TR/vc-data-model-2.0) | [BitStringStatusList](https://www.w3.org/TR/vc-bitstring-status-list/)   | Enveloped proofs [using JWT](https://www.w3.org/TR/vc-jose-cose/#with-jose) | Ignore `ttl`, use `validUntil` *). No JWE supported. The concrete signature algorithm is not specified, as implementors are expected to handle all reasonably well-known crypto algorithms like RSA, EC and EdDSA |
+| `vc11-sl2021/jwt` | [VC DataModel 1.1](https://www.w3.org/TR/vc-data-model)     | [StatusList2021](https://www.w3.org/TR/2023/WD-vc-status-list-20230427/) | External proofs using JWT                                                   | --"--                                                                                                                                                                                                             |
```

Comment:
as far as I can see, [`vc-jose-cose`](https://www.w3.org/TR/vc-jose-cose/#example-a-simple-example-of-a-verifiable-credential-secured-with-jose) recommends a slightly different type of JWT than [`vc-data-model`](https://www.w3.org/TR/vc-data-model/#example-usage-of-the-id-property). Latter requires the VC to be in the `vc` claim while the former makes no use of the ordinary JWT claims. Does this require the spec differentiating between two flavors of JWT?

Reply:
You're right - the textual description should be precise enough. The expectation isn't that implementations dissect the `profile` and then permutate the different profile components freely.